pipelines/docker-build-multi-arch-oci-ta/patch.yaml

```diff
@@ -0,0 +1,73 @@
+---
+- op: replace
+  path: /metadata/name
+  value: docker-build-multi-arch-oci-ta
+# Order of Tasks from the base docker-build Pipeline:
+# $ kustomize build pipelines/docker-build-oci-ta | yq ".spec.tasks.[].name" | nl -v 0
+#      0  init
+#      1  clone-repository
+#      2  prefetch-dependencies
+#      3  build-container
+#      4  build-image-index
+#      5  build-source-image
+#      6  deprecated-base-image-check
+#      7  clair-scan
+#      8  ecosystem-cert-preflight-checks
+#      9  sast-snyk-check
+#     10  clamav-scan
+#     11  sbom-json-check
+#     12  apply-tags
+#     13  push-dockerfile
+# change name to build-containers
+# add build-platforms parameter
+# add matrix to run based on build-platforms
+# change build-image-index to use results from build-containers as well as run after it
+# change build-image-index to build image index
+# change JAVA_COMMUNITY_DEPENDENCIES value to "$(tasks.build-containers.results.JAVA_COMMUNITY_DEPENDENCIES[0])"
```

Comment:
This has been the pattern that we have maintained when exploring multi-arch pipelines. It can be revisited when the java build pipeline is reassessed for multi-arch support.